[2024-10-02T19:56:32.782+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-10-02T19:56:32.807+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: decision_tree_classifier_dag.dtc_save_results manual__2024-10-02T19:56:11.585866+00:00 [queued]>
[2024-10-02T19:56:32.820+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: decision_tree_classifier_dag.dtc_save_results manual__2024-10-02T19:56:11.585866+00:00 [queued]>
[2024-10-02T19:56:32.823+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-10-02T19:56:32.843+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): dtc_save_results> on 2024-10-02 19:56:11.585866+00:00
[2024-10-02T19:56:32.856+0000] {standard_task_runner.py:64} INFO - Started process 1825 to run task
[2024-10-02T19:56:32.859+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'decision_tree_classifier_dag', 'dtc_save_results', 'manual__2024-10-02T19:56:11.585866+00:00', '--job-id', '709', '--raw', '--subdir', 'DAGS_FOLDER/***/dags/hw1.py', '--cfg-path', '/tmp/tmp1jdmvirf']
[2024-10-02T19:56:32.862+0000] {standard_task_runner.py:91} INFO - Job 709: Subtask dtc_save_results
[2024-10-02T19:56:32.934+0000] {task_command.py:426} INFO - Running <TaskInstance: decision_tree_classifier_dag.dtc_save_results manual__2024-10-02T19:56:11.585866+00:00 [running]> on host 4b5f1410d111
[2024-10-02T19:56:33.070+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alexvlg34@gmail.com' AIRFLOW_CTX_DAG_OWNER='Alex Trigolos' AIRFLOW_CTX_DAG_ID='decision_tree_classifier_dag' AIRFLOW_CTX_TASK_ID='dtc_save_results' AIRFLOW_CTX_EXECUTION_DATE='2024-10-02T19:56:11.585866+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-10-02T19:56:11.585866+00:00'
[2024-10-02T19:56:33.073+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-10-02T19:56:33.085+0000] {hw1.py:167} INFO - Save metrics...
[2024-10-02T19:56:33.222+0000] {base.py:84} INFO - Using connection ID 's3_connection' for task execution.
[2024-10-02T19:56:33.224+0000] {connection_wrapper.py:388} INFO - AWS Connection (conn_id='s3_connection', conn_type='aws') credentials retrieved from login and password.
[2024-10-02T19:56:33.893+0000] {hw1.py:194} INFO - DONE!!!
[2024-10-02T19:56:33.896+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-10-02T19:56:33.896+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-10-02T19:56:33.908+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=decision_tree_classifier_dag, task_id=dtc_save_results, run_id=manual__2024-10-02T19:56:11.585866+00:00, execution_date=20241002T195611, start_date=20241002T195632, end_date=20241002T195633
[2024-10-02T19:56:33.960+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-10-02T19:56:33.989+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-10-02T19:56:33.993+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
